### ISSUE TITLE
Inconsistent unordered list style

### ISSUE THREAD

**Issue by KillyMXI** [tgreyuk/typedoc-plugin-markdown#218]
I have a class that implements some interface. Generated documentation (the relevant part) looks like this:

```markdown

# Class: Foo

## Implements

* [*IBuilder*](../interfaces/ibuilder.md)

## Table of contents

### Constructors

- [constructor](foo.md#constructor)

### Methods

- [build](foo.md#build)

...
```

This triggers [MD004 - Unordered list style](https://github.com/DavidAnson/markdownlint/blob/v0.23.1/doc/Rules.md#md004) markdownlint rule due to inconsistent list item markings.

I think it worth to ensure the tool consistently outputs the same symbol for every list.

I believe hyphen (`-`) is the generally preferred symbol but I wouldn't mind if there is a good reason to go with a different one.

**Comment by tgreyuk** [comment]
Thanks - no there should be consistency I will fix it. Would be a good idea to introduce markdownlint into tests too.

**Comment by tgreyuk** [comment]
fix in typedoc-plugin-markdown@3.8.0 .. Also linting all the test stubs.

**Comment by KillyMXI** [comment]
Awesome! Thank you.
